src/Agent.Worker/Extensions/VariablesExtension.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.TeamFoundation.DistributedTask.WebApi;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Extensions
+{
+    public static class VariablesExtension
+    {
+        public static Dictionary<string, VariableValue> ToWebApiVariables(this IEnumerable<Variable> outputVariables)
```

Comment:
Call it `JobCompletedEventOutputVariables`, also do we really need a extension? since it's only called from jobrunner.cs